### PR TITLE
[ini_export] Fix LR_scheduler ini export

### LINKED ISSUE #2296: [ini export] the learning rate scheduler has a problem of being exported
Currently, reading with ini format works normally, **but the results are different when exporting.**
In particular, the learning rate scheduler has a problem of being exported in a completely different way than intended.
Below is the difference when the mnist.ini file is imported and exported to the mnist_model.ini file again.

original
```.ini
[Optimizer]
Type = adam
beta1 = 0.9                         # beta 1 for adam
beta2 = 0.999                       # beta 2 for adam
epsilon = 1e-7                      # epsilon for adam

[LearningRateScheduler]
type=step
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
```

exported
```.ini
[optimizer]
beta1 = 0.9
beta2 = 0.999
epsilon = 1e-07
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
torch_ref = false
type = adam
```

### PR DESCRIPTION
**Pull request by DonghakPark**
## In this PR

**When exporting the model, the Optimizer and LR Scheduler should be exported separately, but solve the problem that does not**

Now when exporting a model that using LR Scheduler, Optimizer and LR Scheduler are not divided separately, and there is an issue in which the property of LR Scheduler is written in the Property of Optimizer.

So, This make Syntax error when loading the model. and this patch fix this issue.

**Now**
```.ini
[optimizer]
beta1 = 0.9
beta2 = 0.999
epsilon = 1e-07
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
torch_ref = false
type = adam
```

**This fetch**
```.ini
[Optimizer]
Type = adam
beta1 = 0.9                       
beta2 = 0.999                   
epsilon = 1e-7                  

[LearningRateScheduler]
type=step
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
```

related issue : close #2296 

""""""""""""""""""""""""""""""""""""""""""""""
**Self evaluation:**
Build test: [x]Passed [ ]Failed [ ]Skipped
Run test: [x]Passed [ ]Failed [ ]Skipped_
""""""""""""""""""""""""""""""""""""""""""""""

Signed-off-by: Donghak PARK <donghak.park@samsung.com>

### CODE REVIEW

**Review by Hyoungsukim — APPROVED** [review]
lgtm

**Review by djeong20 — APPROVED** [review]
LGTM!

File: nntrainer/models/neuralnet.cpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ NeuralNetwork::NeuralNetwork(AppContext app_context_) :
 
 int NeuralNetwork::loadFromConfig(const std::string &config) {
   if (loadedFromConfig == true) {
-    ml_loge("cannnot do loadFromConfig twice");
+    ml_loge("can not do loadFromConfig twice");
```
Comment (wooksong) [inline nntrainer/models/neuralnet.cpp:104]: Is this change made on purpose?

Comment (DonghakPark) [reply nntrainer/models/neuralnet.cpp:104]: Yes, I found a typo while modifying the code.

Comment (wooksong) [reply nntrainer/models/neuralnet.cpp:104]: @DonghakPark https://dictionary.cambridge.org/us/dictionary/english/cannot

[Cannot ](https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference) is far more common and is therefore recommended.

File: nntrainer/compiler/ini_interpreter.cpp [hunk 2]
```diff
@@ -274,7 +274,6 @@ void IniGraphInterpreter::serialize(const GraphRepresentation &representation,
 
     IniSection s = IniSection::FromExportable(ln->getName(), *ln);
     s.setEntry("type", ln->getType());
-
     sections.push_back(s);
```
Comment (myungjoo) [inline nntrainer/compiler/ini_interpreter.cpp:277]: Please do not add  a style fix in a bugfix commit. Remove this file change from the commit.
You will confuse yourself of the future with this commit if you track git log based on filenames in the future.

**Review by myungjoo — COMMENTED** [review]
Remove ini_interpreter.cpp change from this commit.
Do not change ```cannot``` to ```can not```.

Then, it is good to go.

**Review by myungjoo — APPROVED** [review]

**Review by lhs8928 — APPROVED** [review]
LGTM

**Comment by DonghakPark** [comment]
I will reopen PR After refactoring export feature
